src/wizard/examples/wizard.js

```diff
@@ -0,0 +1,309 @@
+  * @param {string=} contentHeight The height the wizard content should be set to. This is used ONLY if the stepClass is not given. This defaults to 300px if the property is not supplied.
+  * @param {boolean=} hideIndicators  Hides the step indicators in the header of the wizard
+  * @param {string=} currentStep The current step can be changed externally - this is the title of the step to switch the wizard to
+  * @param {string=} cancelTitle The text to display on the cancel button
+  * @param {string=} backTitle The text to display on the back button
+  * @param {string=} nextTitle The text to display on the next button
+  * @param {function(step)=} backCallback Called to notify when the back button is clicked
+  * @param {function(step)=} nextCallback Called to notify when the next button is clicked
+  * @param {function()=} onFinish Called to notify when when the wizard is complete.  Returns a boolean value to indicate if the finish operation is complete
+  * @param {function()=} onCancel Called when the wizard is canceled, returns a boolean value to indicate if cancel is successful
+  * @param {boolean} wizardReady Value that is set when the wizard is ready
+  * @param {boolean=} wizardDone  Value that is set when the wizard is done
+  * @param {string} loadingWizardTitle The text displayed when the wizard is loading
+  * @param {string=} loadingSecondaryInformation Secondary descriptive information to display when the wizard is loading
+  * @param {boolean=} embedInPage Value that indicates wizard is embedded in a page (not a modal).  This moves the navigation buttons to the left hand side of the footer and removes the close button.
+  * @param {function(step, index)=} onStepChanged Called when the wizard step is changed, passes in the step and the step index of the step changed to
+  * @deprecated {string} title The wizard title displayed in the head (use wizardTitle instead)
+  * @example
+  <example module="patternfly.wizard" deps="patternfly.form">
+  <file name="index.html">
+    <div ng-controller="WizardModalController">
+      <button ng-click="openWizardModel()" class="btn btn-default">Launch Wizard</button>
+    </div>
+  </file>
+  <file name="wizard-container.html">
+  <pf-wizard wizard-title="Wizard Title"
+    wizard-ready="deployProviderReady"
+    on-finish="finishedWizard()"
+    on-cancel="cancelDeploymentWizard()"
+    next-title="nextButtonTitle"
+    next-callback="nextCallback"
+    back-callback="backCallback"
+    wizard-done="deployComplete || deployInProgress"
+    sidebar-class="example-wizard-sidebar"
+    step-class="example-wizard-step"
+    loading-secondary-information="secondaryLoadInformation"
+    on-step-changed="stepChanged(step, index)">
+      <pf-wizard-step step-title="First Step" substeps="true" step-id="details" step-priority="0" show-review="true" show-review-details="true">
```

Comment:
@jeff-phillips-18 feel a little silly asking this, but was trying to get the uib-tooltip to show up on the buttons... 

Can't seem to get them to load after editing the example under ```src/wizard/examples/wizard.js``` and adding the following attributes to "pf-wizard-step"s like...
```html
<pf-wizard-step step-title="First Step" prev-tooltip="'1 PRIOR TIPS'" next-tooltip="'1 CHECK MY TIPS'" substeps="true"... />

<pf-wizard-step step-title="Second Step" prev-tooltip="'2 PRIOR TIPS'" next-tooltip="'2 CHECK MY TIPS'" substeps="false"... />

<pf-wizard-step step-title="Review" prev-tooltip="'3 PRIOR TIPS'" next-tooltip="'3 CHECK MY TIPS'" substeps="t..." />
```

Is there something I'm missing, or maybe a conflicting that needs to be removed to get them to load? If so then an example update might be in order.

Reply:
Hmmm.... seems they are not working with or without these changes. I'll take a look.